src/main/java/com/synopsys/integration/blackduck/codelocation/signaturescanner/CodeLocationWaitJobTask.java

```diff
@@ -93,7 +93,7 @@ private int retrieveCompletedCount(UserView userView, NotificationTaskRange noti
         Map<String, String> foundCodeLocations = codeLocationViews
                 .stream()
                 .filter(codeLocationView -> codeLocationNames.contains(codeLocationView.getName()))
-                .collect(Collectors.toMap(codeLocationView -> codeLocationView.getHref().get(), CodeLocationView::getName));
+                .collect(Collectors.toMap(codeLocationView -> codeLocationView.getHref().get().string(), CodeLocationView::getName));
```

Comment:
Not a big fan of the size of this method chain being inline AND in a lambda. Let's put it in a private method and handle the `Optional` correctly (even if it just means calling `.orElseThrow()`).